internal/observability/metrics/attributes.go

```diff
@@ -1,18 +1,5 @@
 package metrics
```

Comment:
I think this file should move up one level and be at the `internal/observability` level instead of `internal/observability/metrics` that way we can use shared attributes in traces as well.

Reply:
moved